homeassistant/components/blebox/climate.py

```diff
@@ -21,6 +21,19 @@
 
 SCAN_INTERVAL = timedelta(seconds=5)
 
+BLEBOX_TO_HVACMode = {
```

Comment:
```suggestion
BLEBOX_TO_HVACMODE = {
```
Constants should be all caps.